### ISSUE TITLE
[문의] 학습 과정에서 loss가 줄어들지 않는 문제

### ISSUE THREAD

**Issue by favorite-name** [MrBananaHuman/KorAlbert#3]
안녕하세요 이전에 문의 드렸던 학생입니다!
공유해주신 리소스로 학습 실험을 하고 있는데 시간이 지나도 loss가 감소되지 않는 현상이 있어 열어보니 prediction을 위한 output 부분에서(마지막 레이어 값) 모든 값이 똑같이 나오는 것 같습니다

아래는 batch를 4로 했을 때 모습인데요, start_logit과 end_logit이 모든 지점에서 값이 똑같아 이를 합치면 결국 loss가 계속 동일하게 나옵니다. 혹시 무슨 문제인지 아시는지 여쭤보고 싶습니다ㅜㅜ
(아래 모습은 step이 최소 몇천은 돌아간 상태입니다)

올려주신 코드로도 돌려보고, 코드베이스로 사용하신 https://github.com/brightmart/albert_zh 여기에서 클린하게 다시 내려받아서도 돌려보았는데 동일한 loss 문제가 발생합니다.
원본 google ALBERT 코드로도 베이스를 변경해보았는데 거기서는 리소스 로드 과정에서 shape가 맞지 않는다고 오류가 발생하였습니다.

좋은 리소스 공유 감사드리고 위 현상에 대해 혹시 답을 알려주신다면 큰 도움이 될 것 같습니다. 감사합니다!

```
# 전체 레이어 결과값
all_layers =
[[ 0.07039269  0.6399561   1.5619121  ...  0.21054047  1.0778954 -0.08915286]
 [ 0.07039303  0.6399559   1.5619136  ...  0.21053928  1.0778956 -0.0891524 ]
 [ 0.07039287  0.6399561   1.5619121  ...  0.21054018  1.0778949  -0.0891531 ]
 ...
 [ 0.0703932   0.639956    1.5619136  ...  0.2105393   1.0778955  -0.08915246]
 [ 0.07039299  0.6399562   1.5619143  ...  0.21054007  1.0778948  -0.0891528 ]
 [ 0.07039329  0.6399561   1.5619155  ...  0.21053907  1.077894  -0.08915255]]

# 위 값 중 마지막 레이어 값을 뽑은 것
output_orig =
[[[ 0.05987604  0.76085603  1.50679    ...  0.20458442  0.9688564 -0.06451298]
  [ 0.05987603  0.76085514  1.5067904  ...  0.20458388  0.9688564   -0.06451317]
  [ 0.05987597  0.76085573  1.5067883  ...  0.20458487  0.96885574   -0.06451293]
  ...
  [ 0.05987606  0.7608558   1.5067884  ...  0.20458457  0.9688561   -0.06451259]
  [ 0.05987595  0.7608557   1.50679    ...  0.20458366  0.9688564   -0.06451262]
  [ 0.05987628  0.7608561   1.5067887  ...  0.20458429  0.96885633   -0.06451321]]

 [[ 0.07039269  0.6399561   1.5619121  ...  0.21054047  1.0778954  -0.08915286]
  [ 0.07039303  0.6399559   1.5619136  ...  0.21053928  1.0778956   -0.0891524 ]
  [ 0.07039287  0.6399561   1.5619121  ...  0.21054018  1.0778949   -0.0891531 ]
  ...
  [ 0.0703932   0.639956    1.5619136  ...  0.2105393   1.0778955   -0.08915246]
  [ 0.07039299  0.6399562   1.5619143  ...  0.21054007  1.0778948   -0.0891528 ]
  [ 0.07039329  0.6399561   1.5619155  ...  0.21053907  1.077894   -0.08915255]]

 [[ 0.15624668 -0.24600373  0.2972823  ... -2.4192111   0.18228768  -0.7852281 ]
  [ 0.1562461  -0.24600412  0.2972818  ... -2.419213    0.18228744   -0.78522813]
  [ 0.15624642 -0.2460039   0.2972828  ... -2.4192128   0.18228723   -0.7852283 ]
  ...
  [ 0.15624651 -0.24600315  0.29728204 ... -2.4192114   0.18228772   -0.78522766]
  [ 0.15624681 -0.24600457  0.2972822  ... -2.4192116   0.18228708   -0.7852288 ]
  [ 0.15624644 -0.24600354  0.29728183 ... -2.4192116   0.18228796   -0.7852286 ]]

 [[ 0.13948305 -0.17706998  0.24214134 ... -2.443108    0.14445977  -0.7748494 ]
  [ 0.139483   -0.17707013  0.2421417  ... -2.4431098   0.14446034   -0.7748503 ]
  [ 0.13948287 -0.17707007  0.24214117 ... -2.4431088   0.14446042   -0.77484995]
  ...
  [ 0.13948296 -0.17707002  0.24214104 ... -2.4431098   0.14445992   -0.774849  ]
  [ 0.13948284 -0.17706938  0.24214178 ... -2.4431093   0.14446068   -0.77484936]
  [ 0.13948292 -0.17706937  0.2421414  ... -2.4431093   0.14445993   -0.7748499 ]]],

# 시작 지점 확률(가로 한 줄이 example 1개)
start_logits = 
[[-1.0417409  -1.0417408  -1.0417409  ... -1.0417404  -1.0417408 -1.0417407 ]
 [-1.104904   -1.104904   -1.1049042  ... -1.1049039  -1.1049044  -1.1049042 ]
 [-0.10860953 -0.10861009 -0.10860971 ... -0.10861006 -0.10860991  -0.1086103 ]
 [-0.0719251  -0.0719251  -0.07192543 ... -0.07192445 -0.07192457  -0.07192466]]

# 끝 지점 확률(가로 한 줄이 example 1개)
end_logits =
[[-0.12866224 -0.12866268 -0.1286616  ... -0.12866202 -0.128662 -0.12866224]
 [-0.1294852  -0.12948444 -0.12948483 ... -0.12948498 -0.1294846  -0.1294843 ]
 [ 1.1706426   1.1706421   1.1706426  ...  1.1706421   1.1706423  1.1706425 ]
 [ 1.1731018   1.1731015   1.1731013  ...  1.1731017   1.1731012  1.1731013 ]]

# 시작 지점 loss
start_loss = 5.8277526

# 끝 지점 loss
end_loss = 5.827752

# 최종 loss
loss = 6.0603695
```
